README.md

```diff
@@ -1,192 +1,165 @@
-3.	Create a new MySQL Connection to the database.
-Note: Make sure to keep track of the username, password, url, and port you used as you will need this later.
-4.	From the MAT Code base, find the `mat_schema.sql` file and then execute the script in the database that was just created.
-
-### Configure Files for Development
-Changes will need to be made to some of the files in the code base to allow the MAT to connect to the MySQL Database.
-
-#### add `mat/war/WEB-INF/jetty-env.xml`
-```xml
-<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure.dtd">
-<Configure id="Server" class="org.eclipse.jetty.webapp.WebAppContext">
-<New id="MatJndi" class="org.eclipse.jetty.plus.jndi.Resource">
-  <Arg></Arg>
-  <Arg>java:/comp/env/jdbc/mat_app_tomcat</Arg>
-  <Arg>
-    <New class="com.mysql.jdbc.jdbc2.optional.MysqlConnectionPoolDataSource">
-        <Set name="User">username</Set>
-        <Set name="Password">password </Set>
-        <Set name="DatabaseName">schemaname</Set>
-        <Set name="ServerName">serveraddress</Set>
-        <Set name="PortNumber">port</Set>
-      </New>
-    </Arg>
-</New>
-</Configure>
-```
-If you get exception as below `'com.google.gwt.dev.shell.jetty.JettyLauncher.WebAppContextWithReload'` is not of type `'org.eclipse.jetty.webapp.WebAppContext'` then you will need to find a way to load the jetty class instead of `gwt-dev.jar`. One way to make it work is to change the `gwt-dev` dependency in the `pom.xml` to be `provided` rather than `compile`. If you do this, then when running `mvn clean install` you will have to change it back.
-```xml
-<dependency>
-    <groupId>com.google.gwt</groupId>
-    <artifactId>gwt-dev</artifactId>
-    <version>${gwtVersion}</version>
-    <scope>provided</scope>
-    <exclusions>
-        <exclusion>
-            <artifactId>commons-collections</artifactId>
-            <groupId>commons-collections</groupId>
-        </exclusion>
-        <exclusion>
-            <groupId>org.eclipse.jetty</groupId>
-            <artifactId>apache-jsp</artifactId>
-        </exclusion>
-    </exclusions>
-</dependency>
+- Install MySQL (MAT currently has been tested with MySQL Community Version 8.0+) available from [MySQL] (https://dev.mysql.com/downloads/mysql/)
+- Run the MySQL community server installer for your operating system and the MySQL workbench (which comes with the download).
+- For Mac:
+- Go to System Preferences/ MY SQL after installing.
+- Click Initialize Database.
+- Enter a passowrd and click on Use Legacy Password Encryption. (Remember the username/pwd you will need these in future steps.)
+- Start the MYSQL Db.
+- In MeasureAuthoringTool/pom.xml I had to change the driver to match the community version I downloaded. I am always careful not to commit this change.
+  
 ```
+      <dependency>
+          <groupId>mysql</groupId>
+          <artifactId>mysql-connector-java</artifactId>
```

Comment:
I think this trick is needed only if a developer uses MySQL ver  <8.
If you install recommended MySQL 5.7, then this is not needed.